packages/cli/src/commands/election/vote.ts

```diff
@@ -15,7 +15,7 @@ export default class ElectionVote extends BaseCommand {
       description: "ValidatorGroup's address",
       required: true,
     }),
-    value: Flags.string({ description: 'Amount of Gold used to vote for group', required: true }),
+    value: Flags.string({ description: 'Amount of CELO used to vote for group', required: true }),
```

Comment:
use `CustomFlags.bigint`

Reply:
will do when i convert file to viem. started on it just not this pr